batch/batch/front_end/front_end.py

```diff
@@ -210,6 +210,82 @@ async def _handle_api_error(f, *args, **kwargs):
         raise e.http_response()
 
 
+async def _query_batch_jobs_for_billing(request, batch_id):
+    db = request.app['db']
+
+    # batch has already been validated
+    where_conditions = ['(jobs.batch_id = %s)']
+    where_args = [batch_id]
+
+    last_job_id = request.query.get('last_job_id')
+    limit = min(300, int(request.query.get('limit', 300)))
```

Comment:
Might be better to raise an error if the input parameter is out of range?

Reply:
Thanks! I've caught and raised a ValueError, I've opened up the limit to more values (and the database will throw an error if it's invalid too).